app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/ui/activities/PlaceMealActivity.java

```diff
@@ -95,20 +93,12 @@ public void onClick(View v) {
         switch (v.getId()) {
             case R.id.addPictureButton:
                 Intent galleryIntent = new Intent(Intent.ACTION_PICK, MediaStore.Images.Media.EXTERNAL_CONTENT_URI);
+                //@TODO for sprint 9 I (Raed) will change the upload picture to be abstract and not deprecated
```

Comment:
I can help you with that, we have the profileEditingActivity and the signup acitvity that have a startActivityforresult that is not deprecated